python/src/DeconditionedDistribution.i

```diff
@@ -0,0 +1,11 @@
+// SWIG file DeconditionedDistribution.i
+
+%{
+#include "openturns/DeconditionedDistribution.hxx"
+%}
+
+%include DeconditionedDistribution_doc.i
+
+%copyctor OT::DeconditionedDistribution;
+
```

Comment:
lets use a python-only alias so the c++ definition can be dropped, such as:
%pythoncode %{
def ConditionalDistribution(conditionedDistribution, conditioningDistribution, linkFunction=None):
    openturns.common.Log.Warn('class ConditionalDistribution is deprecated in favor of DeconditionedDistribution')
    if linkFunction is None:
        linkFunction = openturns.func.IdentityFunction(conditioningDistribution.getDimension())
    return openturns.dist_bundle2.DeconditionedDistribution(conditionedDistribution, conditioningDistribution, linkFunction)
%}
same for Bayes

Reply:
done